clients/gtest/gtest.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
This is the master test YAML file, which can be expanded later to include any number of tests.